modules/mongodb/charts.go

```diff
@@ -0,0 +1,490 @@
+		{ID: "connections.available", Name: "available"},
+	},
+}
+
+var (
+	chartNetwork = module.Chart{
+		ID:    "network",
+		Title: "Network IO",
+		Units: "bytes/s",
+		Fam:   "network",
+		Ctx:   "mongodb.network",
+		Type:  module.Area,
+		Dims: module.Dims{
+			{ID: "network.bytesIn", Name: "Bytes In", Algo: module.Incremental, Mul: -1},
+			{ID: "network.bytesOut", Name: "Bytes Out", Algo: module.Incremental},
+		},
+	}
+	chartNetworkRequests = module.Chart{
+		ID:    "networkRequests",
+		Title: "Network Requests",
+		Units: "requests/s",
+		Fam:   "network",
+		Ctx:   "mongodb.networkRequests",
+		Dims: module.Dims{
+			{ID: "network.numRequests", Name: "Requests", Algo: module.Incremental},
+		},
+	}
+)
+
+var chartMemory = module.Chart{
+	ID:    "mem",
+	Title: "Memory",
+	Units: "MiB",
+	Fam:   "memory",
+	Ctx:   "mongodb.memory",
+	Dims: module.Dims{
+		{ID: "mem.resident", Name: "resident"},
+		{ID: "mem.virtual", Name: "virtual"},
+		{ID: "mem.mapped", Name: "mapped"},
+		{ID: "mem.mappedWithJournal", Name: "mapped with journal"},
+	},
+}
+
+var chartPageFaults = module.Chart{
+	ID:    "page_faults",
+	Title: "Page faults",
+	Units: "page faults/s",
+	Fam:   "page_faults",
+	Ctx:   "mongodb.page_faults",
+	Dims: module.Dims{
+		{ID: "extra_info.page_faults", Name: "Page Faults", Algo: module.Incremental},
+	},
+}
+
+var chartAsserts = module.Chart{
+	ID:    "asserts",
+	Title: "Asserts",
+	Units: "asserts/s",
+	Fam:   "asserts",
+	Ctx:   "mongodb.asserts",
+	Type:  module.Stacked,
+	Dims: module.Dims{
+		{ID: "asserts.regular", Name: "regular", Algo: module.Incremental},
+		{ID: "asserts.warning", Name: "warning", Algo: module.Incremental},
+		{ID: "asserts.msg", Name: "msg", Algo: module.Incremental},
+		{ID: "asserts.user", Name: "user", Algo: module.Incremental},
+		{ID: "asserts.tripwire", Name: "tripwire", Algo: module.Incremental},
+		{ID: "asserts.rollovers", Name: "rollovers", Algo: module.Incremental},
+	},
+}
+
+// option charts
+var chartTransactionsCurrent = module.Chart{
+	ID:    "transactionsCurrent",
+	Title: "Current Transactions",
+	Units: "transactions",
+	Fam:   "transactionsCurrent",
+	Ctx:   "mongodb.transactionsCurrent",
+	Dims: module.Dims{
+		{ID: "transactions.currentActive", Name: "current active"},
+		{ID: "transactions.currentInactive", Name: "current inactive"},
+		{ID: "transactions.currentOpen", Name: "current open"},
+		{ID: "transactions.currentPrepared", Name: "current prepared"},
+	},
+}
+
+var chartGlobalLockActiveClients = module.Chart{
+	ID:    "globalLockActiveClients",
+	Title: "Active Clients",
+	Units: "clients",
+	Fam:   "clients",
+	Ctx:   "mongodb.currentQueue",
+	Dims: module.Dims{
+		{ID: "globalLock.activeClients.readers", Name: "readers"},
+		{ID: "globalLock.activeClients.writers", Name: "writers"},
+	},
+}
+
+var chartCollections = module.Chart{
+	ID:    "catalogStats",
+	Title: "Catalog Stats",
+	Units: "objects",
+	Fam:   "catalogStats",
+	Ctx:   "mongodb.catalogStats",
```

Comment:
What is "catalog"? Didn't find it in [the glossary](https://docs.mongodb.com/manual/reference/glossary/) or [serverStatus](https://docs.mongodb.com/manual/reference/command/serverStatus).

Reply:
I think that's a v5 and it's not documented. References can be found in their Jira https://jira.mongodb.org/browse/SERVER-55784?jql=text%20~%20%22catalogStats%22

Reply:
Thanks! How did you find it? 😄 

Ok, "Motivation" section from the Jira ticker

> In order to make more informed product decisions and better support customers feature requests we need to have better insight into our customer usage and behavior. By adding metrics to track usage and behavior ASAP we will be able to not only measure the success of features and inits, but better understand how to develop and prioritize new features related to collection types. 

Do we need these stats? It looks like they (product) need it

> better understand how to develop and prioritize new features related to collection types.